.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
```

Comment:
I prefer to use "semi": true. We can discuss it

Reply:
No need - you are right, I'm agree with you